### PR TITLE
added defaults to user input

### PR DESCRIPTION
**Pull request by hakanErgin**

### CODE REVIEW

File: utils/polygonRequest.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import json
-import re
 import requests
 import matplotlib.pyplot as plt
 
 from typing import Union, Dict
-from pyproj import Proj, transform, Transformer
```
Comment (hakanErgin) [inline utils/polygonRequest.py:7]: these 2 were not being used

File: utils/polygonRequest.py [hunk 2]
```diff
@@ -28,31 +31,7 @@ def getJsonInfo(self) -> Union[float, float, Dict]:
 
         # We do the request of the address of the building we want to visualize
         # Nollekensstraat 15 as default address located into the split tile 212
-
```
Comment (hakanErgin) [inline utils/polygonRequest.py:31]: this block moved to its own file

File: utils/polygonRequest.py [hunk 3]
```diff
@@ -16,6 +15,10 @@ class PolygonRequest:
     polygon_format = '&format=jsonv2&polygon_geojson=1'
     API_url = 'https://nominatim.openstreetmap.org/search?q='
 
+    # transformer class is instantiated only once
+    transformer = Transformer.from_crs("EPSG:4326", "EPSG:31370", always_xy=True)
```
Comment (hakanErgin) [inline utils/polygonRequest.py:18]: this is where the magic happens

File: utils/polygonRequest.py [hunk 4]
```diff
@@ -16,6 +15,10 @@ class PolygonRequest:
     polygon_format = '&format=jsonv2&polygon_geojson=1'
     API_url = 'https://nominatim.openstreetmap.org/search?q='
```
Comment (kaygu) [inline utils/polygonRequest.py]: concat *API_url* & *polygon_format* to same string 
like this :
API_URL = 'https://nominatim.openstreetmap.org/search?format=jsonv2&polygon_geojson=1&q='

Comment (kaygu) [reply utils/polygonRequest.py]: and thus remove *self.polygon_format* line 37, since all the url arguments would be at the beginning of the string

**Pull request merged**